x-pack/plugins/apm/server/routes/agent_explorer/get_agent_url_repository.ts

```diff
@@ -0,0 +1,52 @@
+import { AgentName, isElasticAgentName } from "@kbn/apm-plugin/typings/es_schemas/ui/fields/agent";
+
+const agentsRepoName: Partial<Record<AgentName, string>> = {
+  'go': 'apm-agent-go',
+  'java': 'apm-agent-java',
+  'js-base': 'apm-agent-rum-js',
+  'iOS/swift': 'apm-agent-ios',
+  'rum-js': 'apm-agent-rum-js',
+  'nodejs': 'apm-agent-nodejs',
+  'python': 'apm-agent-python',
+  'dotnet': 'apm-agent-dotnet',
+  'ruby': 'apm-agent-ruby',
+  'php': 'apm-agent-php',
+  'android/java': 'apm-agent-android',
+  'opentelemetry/cpp': 'opentelemetry-cpp',
+  'opentelemetry/dotnet': 'opentelemetry-dotnet',
+  'opentelemetry/erlang': 'opentelemetry-erlang',
+  'opentelemetry/go': 'opentelemetry-go',
+  'opentelemetry/java': 'opentelemetry-java',
+  'opentelemetry/nodejs': 'opentelemetry-js',
+  'opentelemetry/php': 'opentelemetry-php',
+  'opentelemetry/python': 'opentelemetry-python',
+  'opentelemetry/ruby': 'opentelemetry-ruby',
+  'opentelemetry/swift': 'opentelemetry-swift',
+  'opentelemetry/webjs': 'opentelemetry-js',
+};
+
+export const getAllAgentsName = () =>
+  Object.entries(agentsRepoName).map(([agent, _]) => agent as AgentName);
+
+export const getAgentRepositoryDetails = (
+  agentName: AgentName,
+) => {
+  const user = isElasticAgentName(agentName) ? 'elastic': 'open-telemetry';
```

Comment:
Personally, I would have an explicit check about open telemetry in case another agent is added that is neither (or add this info to `agentsRepoName`)  but it's up to you.